app/Repositories/PromoRepository.php

```diff
@@ -96,6 +96,11 @@ public function getRequestData(array $data)
             $group_reference[$data['data']['accordion_promo_group_id']] = 'accordion_page';
         }
 
+        // If there is an grid custom page field then inject it into the group reference
```

Comment:
Since you have a `GridController` is this necessary to check on every request? The reason accordion is here is because it allows the `ChildPageController` to have an accordion at the bottom of every page. It seems like this feature you have to select the `GridController` anyway so it should be unnecessary the grid globally.

Reply:
This was leftover from an idea I had to include it like accordion that I went back on. Removing!